be/src/storage/tablet_updates.cpp

```diff
@@ -2431,6 +2431,13 @@ Status TabletUpdates::link_from(Tablet* base_tablet, int64_t request_version) {
                      << " tablet:" << _tablet.tablet_id() << " base_tablet:" << base_tablet->tablet_id();
         return Status::InternalError("link_from: max_version < request version");
     }
+    if (this->max_version() >= request_version) {
```

Comment:
If the task is ignored, does the sc failed?

Reply:
Then how to guarantee the task can succeed next time?

Reply:
it succeeds